teachopencadd/talktorials/002_compound_adme/talktorial.ipynb

```diff
@@ -6,11 +6,11 @@
    "source": [
```

Comment:
<ul><li>suggest to rename function: calculate_ro5 -&gt; compliance_to_ro5 or ro5_compliance or 	calculate_ro5_compliance?</li><li>one could also add the parameters and returns of the function</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/TeachOpenCADD/pull/49/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
<p>I settled for <code>calculate_ro5_properties</code>  since we calculate not only the compliance but also the individual properties.</p>